server/ai_client.py

```diff
@@ -109,12 +109,14 @@ def __init__(self, room, nickname, ai_agent_file_name=None, waiting_for_respawn=
 
         self.agent.delivery_zone = self.game.delivery_zone.to_dict()
 
-        # Start the AI thread
+        # Start the AI thread only if not in grading mode
```

Comment:
This comment is slightly misleading since there's no if statement here. Maybe say: `Start the AI thread. This code doesn't get called in grading mode` or something like that?